packages/vats/src/core/manifest.js

```diff
@@ -1,5 +1,6 @@
 // @ts-check
 const SHARED_BOOTSTRAP_MANIFEST = harden({
+  makeCoreEval: true,
```

Comment:
is `makeCoreEval` useful on the sim-chain?

Reply:
I've renamed it to bridgeCoreEval and moved it to just the chain manifest.